### PR TITLE
#27 Additional properties for quality matrix

### PR DESCRIPTION
**Pull request by RMeissnerCC**
From mds_oeh meta data set

### CODE REVIEW

File: src/app/api/quality_matrix/quality_matrix.py [hunk 1]
```diff
@@ -49,8 +49,175 @@ def all_sources() -> dict[str, int]:
...
+        "virtual:editorial_file_type",
+        "virtual:editorial_edit_type",
+        "virtual:editorial_filter",
+        "virtual:editorial_status",
+        "virtual:publish_location",
+        "virtual:data_privacy",
+        "virtual:newsletter",
+        "virtual:email",
+        "ccm:lifecyclecontributer_publisher",
+        "ccm:replicationsource",
```
Comment (MRuecklCC) [inline src/app/api/quality_matrix/quality_matrix.py:78]: why are some of the entries here duplicated? Is that intentional?

**Review by MRuecklCC — COMMENTED** [review]
The duplication of properties seems a mistake to me - but otherwise its good to have a list in code where one can see what is dealt with :)

File: src/app/api/quality_matrix/quality_matrix.py (hunk 1 above)
Comment (RMeissnerCC) [reply src/app/api/quality_matrix/quality_matrix.py:77]: Oh, good spotting. Not intended - bizarre. Fixed it

File: src/app/api/quality_matrix/quality_matrix.py [hunk 2]
```diff
@@ -49,8 +49,161 @@ def all_sources() -> dict[str, int]:
...
+        "ccm:general_identifier",
+        "ccm:wwwurl",
+        "ccm:oeh_digitalCompetencies",
+        "ccm:oeh_languageLevel",
+        "cclom:aggregationlevel",
+        "ccm:curriculum",
+        "ccm:lifecyclecontributer_author",
+        "ccm:author_freetext",
+        "ccm:custom_license",
```
Comment (MRuecklCC) [inline src/app/api/quality_matrix/quality_matrix.py:86]: [another](https://tenor.com/view/another-thor-chrishemsworth-gif-4736083)

Comment (RMeissnerCC) [reply src/app/api/quality_matrix/quality_matrix.py]: -_- +#"§$"§$ I recreated the list from a set() - no more duplicates :D

**Review by MRuecklCC — APPROVED** [review]

**Comment by MRuecklCC** [comment]
As it is not merged yet: It would be nie to actually have the list sorted in alphabetical order :P

**Comment by RMeissnerCC** [comment]
> As it is not merged yet: It would be nie to actually have the list sorted in alphabetical order :P

Done - can I merge now? :D

**Pull request merged**